site/kube-addon.md

```diff
@@ -77,10 +77,21 @@ definition](http://kubernetes.io/docs/api-reference/extensions/v1beta1/definitio
 
 ###<a name="blocked-connections"></a> Troubleshooting Blocked Connections
 
-If you suspect that legitimate traffic is being blocked by the Weave Network Policy Controller, check the `weave-npc` container's logs:
+If you suspect that legitimate traffic is being blocked by the Weave Network Policy Controller, check the `weave-npc` container's logs.
+
```

Comment:
If you suspect that legitimate traffic is being blocked by the Weave Network Policy Controller, the first thing to do is to check....